### PR TITLE
Rebrand Proposal details

### LINKED ISSUE #461: Rebrand DAO Dashboard - Proposals table
Update Proposals table based on the [new designs]. The table is used for active proposals on the Governance page and for past proposals on the History page. 

⚠️Do not add pagination to the table - contact @mcoetzee who already did some job on that.

(https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=9057-88620&node-type=section&m=dev) 

Image

### PR DESCRIPTION
**Pull request by peterjurco**
Closes #461 

### What does this change?
- Applies new designs on the Proposal detail page
- Adds testing capabilities

### How to test it?
- Uncomment the line 98 in `src/chain-data/context.tsx`
- Same as in #499 - you can change the `test-proposals.json` file to simulate different proposal states

### Screenshot

image

### CODE REVIEW

File: src/pages/proposal-commons/proposal-details/proposal-details.module.scss
Comment (Anboias) [inline src/pages/proposal-commons/proposal-details/proposal-details.module.scss:5]: Earlier today I pushed the PR with the Modals, which removes the `Button` component wrapper. Please lose this `button` after you incorporate the new changes.

**Review by Anboias — CHANGES_REQUESTED** [review]
Nice job 👌

File: src/pages/proposal-commons/proposal-details/proposal-details.module.scss
Comment (peterjurco) [reply src/pages/proposal-commons/proposal-details/proposal-details.module.scss]: Thanks for thinking about this 🙏 I have rebased this PR and removed the wrapper.

**Review by Anboias — APPROVED** [review]
LGTM

**Comment by peterjurco** [comment]
After the merge of #500 and further thoughts I switched all success and error icons into react components.

**Review by mcoetzee — APPROVED** [review]
Just got minor comments, approving so long.

**Pull request merged**
